src/token/IERC20.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Can we rename this file to `ERC20Like.sol`?

Maybe we should actually create an interfaces folder. But let's decide how to organize that properly later